packages/discovery-provider/src/api/v1/tracks.py

```diff
@@ -757,7 +609,10 @@ def get(self, track_id):
                 cached_content_node, path, skip_check
             )
             if stream_url:
-                return stream_url
+                if no_redirect:
+                    return success_response(stream_url)
```

Comment:
What does this response look like? Is it an object with `data` or does it just return a string?
Also, can you regenerate the SDK so the `no_redirect` field shows up in the request parameters there? Ultimately we will migrate off of the APIClient method onto the SDK version.

Reply:
Response with `data` as the value. [Sample](https://discoveryprovider3.staging.audius.co/v1/tracks/zEoR8Aw/stream-url?user_data=Gated%20content%20user%20signature%20at%201718292975051&user_signature=0xf1755e14393b5ad249650dcfe0c5363f26f168a63c147276ae3d0b51ac339a1e3d263ffa05ec33371f2e425c38272ea228eae3bf5507f9f3b4d6c0b527fb93ca1c) if you're curious